### PR TITLE
fix #834: [ES] Support 'color' template

### LINKED ISSUE #834: [ES] Support 'color' template
- Wiktionary page: https://es.wiktionary.org/w/index.php?title=agamuzado

Wikicode:
```
{{color|#DDB88E|espacio=6}}
```

Output:
```

```

Expected:
```
[RGB #DDB88E]
```

---

Model link, if any: https://es.wiktionary.org/wiki/Plantilla:color

Also check-word needs to be modified.

### PR DESCRIPTION
**Pull request by lasconic**
Also improve test coverage.
Improve a couple of other ES templates

- [x] Fixes #834
- [x] Tests added/updated.
- [x] `./check.sh` run and eventual issues fixed.
- [x] Updated CONTRIBUTORS.md, if not already done in the past.

### CODE REVIEW

**Comment by BoboTiG** [comment]
I will merge when "code quality" will pass ;)

File: wikidict/lang/fr/__init__.py [hunk 1]
```diff
@@ -730,6 +730,9 @@ def last_template_handler(
         >>> last_template_handler(["nom langue", "gcr"], "fr")
         'créole guyanais'
 
+        >>> last_template_handler(["zh-l", "餃子/饺子", "jiǎozi", "jiaozi bouillis"], "fr")
```
Comment (BoboTiG) [inline wikidict/lang/fr/__init__.py:733]: 👍 :)

File: wikidict/lang/en/__init__.py [hunk 2]
```diff
@@ -178,6 +178,9 @@ def last_template_handler(
         >>> last_template_handler(["standard spelling of", "en", "enroll"], "en")
         '<i>Standard spelling of</i> <b>enroll</b>.'
 
+        >>> last_template_handler(["zh-m", "痟", "tr=siáu", "mad"], "en")
```
Comment (BoboTiG) [inline wikidict/lang/en/__init__.py:181]: 👍 :)

**Comment by BoboTiG** [comment]
BTW, when the CI is green, I think you could merge your own PRs :)
We both have a little amount of free time, so waiting on each other for a review when it is not necessary is a loss of time and energy. Let's ping us only when needed by requesting manually a review, WDYT?

**Comment by BoboTiG** [comment]
Last thing :)
I added the sponsor button. I wanted to add you in the list of people, but you are not part of the GitHub sponsoring program. If you plan to integrate the program, just update https://github.com/BoboTiG/ebook-reader-dict/blob/d744dcdf6200b10b834dfdf2920bb1f58a5df90f/.github/FUNDING.yml#L3 ;)

**Comment by lasconic** [comment]
Github sponsoring: I don't plan to join but thanks for asking
Merging, ok I will.

**Pull request merged**